client/state/data-layer/wpcom-http/utils.js

```diff
@@ -83,7 +88,7 @@ export const makeParser = ( schema, schemaOptions = {}, transformer = identity )
 		try {
 			return transformer( data );
 		} catch ( e ) {
-			throw new TransformerError( e, transformer, data );
+			throw new TransformerError( e, data, transformer );
```

Comment:
what's the swap of arguments here for?

Reply:
I wanted to make the `transformer` parameter more optional. I started to throw `TransformerError` from a custom transformer (see `data-layer/wpcom/sites/simple-payments/index.js`) and the pointer to the transformer function is not always available. It can be an inline lambda function etc.

I'm also not sure if the `transformer` function pointer is useful at all -- can we extract any useful info to report from it?

Reply:
not sure - I don't know of any immediate need so let's rip it out - easy enough to add later if we need it